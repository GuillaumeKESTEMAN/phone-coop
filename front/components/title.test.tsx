import React from 'react';
import Title from "./title";
import renderer from 'react-test-renderer';

describe("Title", () => {
  test("should work", () => {
    const rendered = renderer
      .create(<Title level={1}>Title test</Title>)
      .toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
