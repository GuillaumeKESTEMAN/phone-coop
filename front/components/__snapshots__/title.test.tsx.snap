// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should work 1`] = `
<h1
  className="h1"
>
  Title test
</h1>
`;
